src/main/scala/cz/alenkacz/db/postgresscala/PostgresAsyncDbValue.scala

```diff
@@ -35,43 +35,46 @@ class PostgresAsyncDbValue(value: Any) extends DbValue {
     case _ => throw new IllegalArgumentException("Unsupported datetime type - expecting DateTime or LocalDatetime from joda library")
   }
 
-  override def time: Time = value.asInstanceOf[Time]
+  override def time: Time = {
+    val yodaTime = value.asInstanceOf[org.joda.time.LocalTime]
+    Time.valueOf(LocalTime.of(yodaTime.getHourOfDay, yodaTime.getMinuteOfHour, yodaTime.getSecondOfMinute))
+  }
 
-  override def bytes: Array[Byte] = value.asInstanceOf[Array[Byte]]
+  override def bytes: Seq[Byte] = value.asInstanceOf[Array[Byte]].toList
```

Comment:
the reasoning behind toList was that most of the methods return ArrayBuffer which is mutable and I do not want to expose anything mutable

so I just changed it for this byte array